libs/sdk-core/src/lsp.rs

```diff
@@ -23,29 +24,59 @@ pub struct LspInformation {
     pub fee_rate: f64,
     pub time_lock_delta: u32,
     pub min_htlc_msat: i64,
-    pub channel_fee_permyriad: i64,
     pub lsp_pubkey: Vec<u8>,
-    pub max_inactive_duration: i64,
-    pub channel_minimum_fee_msat: i64,
+    pub opening_fee_params_menu: OpeningFeeParamsMenu,
```

Comment:
On second thought there is an advantage for using a struct that holds the list so we can add more fields inside to help the user.
I am OK with keeping the structure but maybe rename it to `opening_fee_options` ?
This is mostly because it is exposed to the binding and reflected in the API.

Reply:
Done with cd431b8957c6c909a3560f97480e4d091c624d60

The struct is also useful because we can have a validation method on it (validating if the list is ordered, etc, as opposed to validating each entry, which we do separately).